Dockerfile

```diff
@@ -3,15 +3,15 @@ ARG BASE_IMAGE
 FROM $BASE_IMAGE
 
 ENV ROOT_PATH /opt/gatling
+ENV HOME=${ROOT_PATH}
 
-COPY docker-entrypoint ${ROOT_PATH}/bin/
+COPY --chmod=775 --chown=1001:0 docker-entrypoint /opt/gatling/bin/
 
-RUN chmod 755 ${ROOT_PATH}/bin/docker-entrypoint && \
-    chmod -R ug=rwx /opt/gatling
-
-WORKDIR ${ROOT_PATH}
+RUN chmod -R ug=rwx /opt/gatling
 
 EXPOSE 9999
 
+USER 1001
```

Comment:
Do we have a `1001` user in `/etc/password`?
(needed with istio and other sidekick managers)

Reply:
Done:

```
$ kubectl exec -it fl-test-1342503487-1 -- bash
bash-5.1$ whoami
gatling
bash-5.1$ cat /etc/passwd
gatling:x:1001:0::/opt/gatling:/bin/bash
```